### DISCUSSION TITLE
Error Reading video stream...

### DISCUSSION THREAD

**Discussion by MonsterFght** [GerryDazoo/Slinger#477] [General]
Slingbox by Gerry has been working perfectly (a little audio lag was hoping to dig around and ask for help there) but now every time I try to connect I get the following trying to connect on local network (v3.08e) via VLC:

Slinginfo  "350/500/M1/M2" 12 30 192.168.1.34 5306 3072 10
8080 Starting Stream for  127.0.0.1:59740
Connecting... ('192.168.1.34', 5306) Control
8080 Selecting Video Source 1
Keycodes= [1, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 24, 25, 26, 27, 28, 29, 30, 31, 33, 35, 36, 38, 39, 40, 41, 42, 45, 46, 53, 62, 63, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.1.34', 5306) Stream
8080 Stream started at 12/26/2022, 14:57:17.624  744 0
.Error Reading video stream
12/26/2022, 14:57:44.313  8080 Stream Stopped Unexpectly. Kicked Off?
8080 Shutting down connections
8080 Logging Out

I have tried different ports other than 8080 thinking my firewall was the issue but still nothing.

**Comment by GerryDazoo** [comment]
you should get the latest 3.08f to start. But that error indicates issues getting packets from the Slingbox. Any lag/sync issues are with your Player. There's nothing I can do about it.

**Comment by MonsterFght** [comment]
Tried it on v3.08f.  Used the default ini file and it worked the first time - I got excited.  I re-tried again and back to the same issue.

Slinginfo  "350/500/M1/M2" 12 30 192.168.1.34 5306 3072 10
8080 Starting Stream for  127.0.0.1:61613
Connecting... ('192.168.1.34', 5306) Control
8080 Selecting Video Source 1
Keycodes= [1, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 21, 24, 25, 26, 27, 28, 29, 30, 31, 33, 35, 36, 38, 39, 40, 41, 42, 45, 46, 53, 62, 63, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 63 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.1.34', 5306) Stream
8080 Stream started at 12/26/2022, 19:30:19.399  744 0
8080 Error Reading video stream
12/26/2022, 19:30:38.525  8080 Stream Stopped Unexpectly. Kicked Off?
8080 Shutting down connections
8080 Logging Out
Streamer:  8080 Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

Is my lag issue due to bandwidth?  Also, how would I be able to "reconfigure" my Slingbox (ie. connect it to another Wi-Fi network)?

**Reply by Fran-Hsu** [reply]
Hi I got the same issue,  the same error log as you posted here.
This issue is gone when I connect the HDMI-out to my TV, so maybe you can check if your HDMI-out cable is connected or not.

**Comment by GerryDazoo** [comment]
Try a different player. VLC issues are the bain of my existence, but it's not the server that is broken.
To your second question, I don't know, I've never configured the WiFi on a Singbox. How did you do it the first time? Try that.

**Comment by MonsterFght** [comment]
For some reason, VLC today has had good times and bad times but glitchy at best  (stream starts and then get "kicked off" with the "8080 Error Reading video stream" error).  I am a NOOB to the other tools like OBS Studio, Ffplayer, and 
Mxplayer (used VLC for years for other things) to see how to connect it to the network stream.  

As for configuring it the first time, I used Slingplayer but now, since the server is gone, I don't see it.  Maybe there is another reader that might know.

**Comment by GerryDazoo** [comment]
So from the latest logs it looks like your slingbox is refusing to play the video stream you have requested. So it doesn't look like a player issue. What type of slingbox are you using? I'm also a little confused are you saying it works sometimes then stops? When it gets into this sate have you tried powering off and on the slingbox? Please also verify there is a valid video signal connected to the slingbox.
PS. Most of the servers are still alive, they've just disabled the ability to watch your box remotely (no place shifting).

**Comment by Toei79** [comment]
> Try a different player. VLC issues are the bain of my existence, but it's not the server that is broken. To your second question, I don't know, I've never configured the WiFi on a Singbox. How did you do it the first time? Try that.

Slinbox app worked with Microsoft  stuff i guess? Idk if I'm right but if yes better a player compatible with like windows media player?

**Comment by robosmith** [comment]
MxPlayer does not seem to have the audio lag issues that VLC has.
I have found that you can set the audio lag value in VLC and -250ms is approximately right.
